src/test/java/reposense/model/RepoConfigurationTest.java

```diff
@@ -185,6 +188,48 @@ public void repoConfig_ignoresStandaloneConfigInCli_success()
         TestUtil.compareRepoConfig(expectedConfig, actualConfig);
     }
 
+    @Test
+    public void repoConfig_ignoresStandaloneConfigInCliOverridesCsv_success()
+            throws HelpScreenException, IOException, ParseException {
+
+        RepoConfiguration repoDeltaExpectedConfig = new RepoConfiguration(
+                new RepoLocation(TEST_REPO_DELTA), "master");
+        repoDeltaExpectedConfig.setFormats(FileType.convertFormatStringsToFileTypes(CLI_FORMATS));
+        repoDeltaExpectedConfig.setStandaloneConfigIgnored(true);
+
+        RepoConfiguration repoCharlieExpectedConfigWithoutIgnoreStandalone = new RepoConfiguration(
+                new RepoLocation(TEST_REPO_CHARLIE));
+        RepoConfiguration repoCharlieExpectedConfigWithIgnoreStandalone = new RepoConfiguration(
+                new RepoLocation(TEST_REPO_CHARLIE));
+        repoCharlieExpectedConfigWithIgnoreStandalone.setStandaloneConfigIgnored(true);
+
+        String inputWithoutIgnoreStandaloneFlag = new InputBuilder().addConfig(IGNORE_STANDALONE_FLAG_OVERRIDE_CSV_TEST)
+            .build();
+        String inputWithIgnoreStandaloneFlag = new InputBuilder().addConfig(IGNORE_STANDALONE_FLAG_OVERRIDE_CSV_TEST)
+            .addIgnoreStandaloneConfig()
+            .build();
+
+        CliArguments argsWithoutIgnoreStandaloneFlag = ArgsParser.parse(
+            translateCommandline(inputWithoutIgnoreStandaloneFlag));
+        CliArguments argsWithIgnoreStandaloneFlag = ArgsParser.parse(
+                translateCommandline(inputWithIgnoreStandaloneFlag));
+
+        List<RepoConfiguration> repoConfigsWithSomeIgnoringStandaloneConfig =
+                RepoSense.getRepoConfigurations((ConfigCliArguments) argsWithoutIgnoreStandaloneFlag);
+        List<RepoConfiguration> repoConfigsWithAllIgnoringStandaloneConfig =
+                RepoSense.getRepoConfigurations((ConfigCliArguments) argsWithIgnoreStandaloneFlag);
```

Comment:
This test case as a whole is confusing as it seems to me that you are combining multiple two sets of test cases into one.

Can we split this TC be split into two? i.e.:
1. RepoConfigs without the CLI ignore standalone config flag --> the column in the CSV file should not be "overriden"
2. RepoConfigs with the CLI ignore standalone config flag --> the flag should take priority and "override" the column in CSV file

Reply:
This makes sense. Thanks for the suggestion @jylee-git 
There already exists a test case for parsing a CSV **without the CLI ignore standalone config flag**. 

However, the CSV used for this test case only has a single repo config.
I think this test CSV should have 2 repo configs - with one repo config ignoring the standalone, and the other not ignoring.